packages/zoe/tools/setup-zoe.js

```diff
@@ -61,12 +61,31 @@ export const setUpZoeForTest = async ({
   );
 
   /**
-   * @param {string} path
+   * @param {object} pathOrExports
    * @returns {Promise<Installation>}
    */
-  const bundleAndInstall = async path => {
-    const bundle = await bundleSource(path);
-    const id = `b1-${path}`;
+  const bundleAndInstall = async pathOrExports => {
+    await null;
+    let id;
+    let bundle;
+    if (typeof pathOrExports === 'string') {
+      const path = pathOrExports;
+      bundle = await bundleSource(path);
+      id = `b1-${path}`;
+    }
+    {
+      assert.equal(
+        Object.prototype.toString.call(pathOrExports),
+        '[object Module]',
+      );
```

Comment:
An equivalent would be to `getOwnPropertyDescriptor(pathOrExports, Symbol.toStringTag)` and verify its value is `Module`